js/src/components/tree-select-control/options.js

```diff
@@ -17,16 +17,18 @@ import { ROOT_VALUE } from './constants';
 /**
  * This component renders a list of options and its children recursively
  *
- * @param {Object} params Component parameters
- * @param {Option[]} params.options List of options to be rendered
- * @param {string[]} params.value List of selected values
- * @param {string[]} params.nodesExpanded List of expanded nodes.
- * @param {Function} params.onChange Callback when an option changes
- * @param {Function} params.onNodesExpandedChange Callback when a node is expanded/collapsed
+ * @param {Object} props Component parameters
+ * @param {Option[]} props.options List of options to be rendered
+ * @param {string[]} props.value List of selected values
+ * @param {string[]} props.nodesExpanded List of expanded nodes.
+ * @param {string} props.filter The filter applied to the Tree Options
```

Comment:
💅  Would it be better to indicate it's an optional prop with a default value?

Reply:
Updated. Please notice I changed also the name to `{boolean} isFiltered ` following other suggestions